src/app/(home)/area/[areaId]/page.tsx

```diff
@@ -35,9 +35,19 @@ const AreaPage = async ({
     },
   });
 
+  const meta = area.meta as {
+    iso3?: string;
+    totalpop?: number;
+    hdi?: number;
+    aggdp_2010?: number;
+    gdppc?: number;
+  };
```

Comment:
Any chance, we can turn this into a proper type? Do we have a type for `area` where we can specify this?